prompt-service/src/unstract/prompt_service/env_manager.py

```diff
@@ -0,0 +1,11 @@
+import os
+from typing import Optional
+
+
+class EnvLoader:
+    @staticmethod
+    def get_env_or_die(env_key: str, default: Optional[str] = None) -> str:
```

Comment:
NIT: We could rename this to `get_env_or_raise_err`

Reply:
@chandrasekharan-zipstack Nothing changed here, Just moved the function here. About the renaming, Both seems same as per the functionality. Will be hard if any changes comes between the merge. So we can do the renaming later.